packages/block-library/src/template-part/edit/index.js

```diff
@@ -154,6 +192,28 @@ export default function TemplatePartEdit( {
 		);
 	}
 
+	const partsAsPatterns = templateParts.map( ( templatePart ) =>
+		mapTemplatePartToBlockPattern( templatePart )
+	);
+
+	const onTemplatePartSelect = ( templatePart ) => {
```

Comment:
This is also defined in packages/block-library/src/template-part/edit/selection-modal.js, but I think they are simple enough that it's ok.